munit-scalacheck/shared/src/main/scala/munit/ScalaCheckSuite.scala

```diff
@@ -29,7 +29,7 @@ trait ScalaCheckSuite extends FunSuite {
   implicit def unitToProp(unit: Unit): Prop = Prop.passed
 
   override def munitTestTransforms: List[TestTransform] =
-    super.munitTestTransforms :+ scalaCheckPropTransform
+    scalaCheckPropTransform +: super.munitTestTransforms
```

Comment:
Feels like the discussions around transform registrations paid off big time. I was a big proponent for side-effecting transforms but that would have been a nightmare to debug for this bug here.